package.json

```diff
@@ -1,9 +1,12 @@
 {
   "name": "token-lists",
-  "version": "1.0.4",
+  "version": "1.1.0",
   "description": "Manages custom token lists for Brave Wallet",
   "dependencies": {
     "@metamask/contract-metadata": "git+https://github.com/MetaMask/contract-metadata.git",
+    "@solana/spl-token-registry": "github:brave/solana-token-list",
+    "nodejs-file-downloader": "^4.9.3",
```

Comment:
Needed this extra dependency to do stuff like retries, guess extension from content disposition, simpler error handling, etc.

Reply:
Just noting that the published token list is generated and so doesn't have this dependency. So it's only used on our CI side for generating the static lists.